vscode/package.json

```
@@ -10,9 +10,8 @@
   "scripts": {
     "dev": "pnpm run build:dev && CODY_FOCUS_ON_STARTUP=1 NODE_ENV=development code --extensionDevelopmentPath=\"$INIT_CWD/vscode\" --disable-extension=sourcegraph.cody-ai --disable-extension=github.copilot --disable-extension=github.copilot-nightly --inspect-extensions=9333 . --goto ./src/logged-rerank.ts:16:5",
     "generate:completions": "ts-node-transpile-only ./src/testutils/cli/run-code-completions-on-dataset.ts",
-    "build": "tsc --build && pnpm run esbuild --minify && vite build --mode production",
-    "build:dev": "tsc --build && concurrently \"pnpm run esbuild --sourcemap\" \"vite build --mode development\"",
-    "prepare": "scripts/download-rg.sh",
+    "build": "scripts/download-rg.sh && tsc --build && pnpm run esbuild --minify && vite build --mode production",
```

```suggestion
    "build": "tsc --build && pnpm run esbuild --minify && vite build --mode production",
```
If we are running the `download-rg.sh` script in the prepublish step, which will be called [every time the extension is packaged](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prepublish-step), does that mean we don't need to rerun the script in the build step?

---

I added it here for local dev. It's a footgun in local dev if you don't have the `rg` binaries installed. Now that it's super quick to run (it doesn't even hit the GitHub API again if you've already downloaded the binaries, so it takes like 5ms), I think it's fine to run it every invocation here. That also helps if we bump the required `rg` version when you are on local dev.